### PR TITLE
Update Kueue's manifests to 0.7.0

### PR DESCRIPTION
**Pull request by PBundyra**

### CODE REVIEW

**Comment by alculquicondor** [comment]
Any changes with respect to the manifests published in the release?

File: tutorials-and-examples/workflow-orchestration/dws-examples/README.md [hunk 1]
```diff
@@ -1,11 +1,20 @@
 # Dynamic Workload Scheduler examples
 
-The repository contains examples on how to use DWS in GKE. More information about DWS is 
+The repository contains examples on how to use DWS in GKE. More information about DWS is
 available [here](https://cloud.google.com/kubernetes-engine/docs/how-to/provisioningrequest).
 
-Files included:
+## Prerequisites:
```
Comment (alculquicondor) [inline tutorials-and-examples/workflow-orchestration/dws-examples/README.md:6]:
```suggestion
## Prerequisites
```

File: tutorials-and-examples/workflow-orchestration/dws-examples/README.md [hunk 2]
```diff
@@ -1,11 +1,20 @@
...
+
+### [Kueue](https://kueue.sigs.k8s.io/)
+To install a released version of Kueue in your cluster, run the following command:
+
+```sh
+VERSION=v0.7.0
+kubectl apply --server-side -f https://github.com/kubernetes-sigs/kueue/releases/download/$VERSION/manifests.yaml
+```
```
Comment (alculquicondor) [inline tutorials-and-examples/workflow-orchestration/dws-examples/README.md:14]: Also link to https://kueue.sigs.k8s.io/docs/installation/ for more configuration options

Comment (PBundyra) [reply tutorials-and-examples/workflow-orchestration/dws-examples/README.md:14]: Done

**Review by alculquicondor — COMMENTED** [review]
/lgtm
/approve

**Review by alculquicondor — APPROVED** [review]

**Pull request merged**